.golangci.yml

```diff
@@ -21,19 +22,17 @@ linters:
     - ineffassign
     - misspell
     - nakedret
+    - nolintlint
     - prealloc
     - revive
-    - exportloopref
     - staticcheck
     - structcheck
     - stylecheck
     - typecheck
     - unconvert
-    - unused
     - unparam
-    - misspell
```

Comment:
this one got deleted. maybe we should still keep it?

Reply:
It isn't deleted, the list was just reordered.